test/e2e/specs/editor/various/preferences.spec.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'preferences', () => {
+	test( 'remembers sidebar dismissal between sessions', async ( {
+		page,
+		admin,
+	} ) => {
+		await admin.createNewPost();
+		// Open by default.
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
+		).toHaveText( 'Post' );
+
+		// Change to "Block" tab.
+		await page.click( 'role=button[name="Block"i]' );
+
+		await expect(
+			page.locator(
+				'.components-button.edit-post-sidebar__panel-tab.is-active'
+			)
```

Comment:
Again I think a role selector can be used for this. It will be different to the one above as it's a different button:
![Screen Shot 2022-09-15 at 5 09 40 pm](https://user-images.githubusercontent.com/677833/190364312-dd809a79-1752-45cc-b4f5-96a93d8ddde5.png)